src/client/components/hazards-list/hazards-list.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <ul class="hazards-list">
+    <li
+      v-for="(hazard, index) in hazards"
+      :key="index"
```

Comment:
dont use index as a key in a v-for loop:
https://vuejs.org/v2/guide/list.html#key